allensdk/brain_observatory/nwb/eye_tracking/extension_builder.py

```diff
@@ -37,6 +38,12 @@ def main():
                 doc='ellipse area',
                 shape=(None, )
             ),
+            NWBDatasetSpec(
+                name='area_raw',
+                dtype='float',
+                doc='ellipse area',
```

Comment:
Can the doc be more descriptive for the `area` type since it now has a different meaning?

e.g. `area_raw`: ellipses areas calculated directly from ellipse fits without regard for likely blinks

Reply:
done